### PR TITLE
Adding Node V22 note for DB2 plug-in

### PR DESCRIPTION
**Pull request by anaxceron**
Describe your pull request here: Address [Issue 3943](https://github.com/zowe/docs-site/issues/3943)

List the file(s) included in this PR: cli-swreqplugins.md, cli-db2plugin.md

After creating the PR, follow the instructions in the comments.

### CODE REVIEW

File: docs/user-guide/image.png
Comment (t1m0thyj) [inline docs/user-guide/image.png:1]: Do we want to add this file to the repo?

File: docs/user-guide/cli-swreqplugins.md [hunk 1]
```diff
@@ -5,10 +5,10 @@ Before installing a Zowe&trade; CLI plug-in, meet the software requirements to r
 :::info Required role: systems administrator
 :::
 
-| Plug-in | Requirements |
+| Plug-in | Requirements!LLL |
```
Comment (adam-wolfe) [inline docs/user-guide/cli-swreqplugins.md:8]: What is `!LLL`?

Comment (anaxceron) [reply docs/user-guide/cli-swreqplugins.md:8]: Was running into errors and added random characters to test local builds. Forgot to remove them lol

File: docs/user-guide/image.png
Comment (anaxceron) [reply docs/user-guide/image.png:1]: There was nothing intentional about that file lol. Removing.

**Review by adam-wolfe — APPROVED** [review]
LGTM, thanks @anaxceron

**Review by t1m0thyj — APPROVED** [review]
LGTM, thanks @anaxceron!

**Pull request merged**